recipes/mdspan/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
```

Comment:
```suggestion
import os

required_conan_version = ">=1.33.0"


```

due to `tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)`